tests/test_requests.py

```diff
@@ -425,7 +453,7 @@ async def app(scope: Scope, receive: Receive, send: Send) -> None:
         # Browsers don't send extra whitespace or semicolons in Cookie headers,
         # but cookie_parser() should parse whitespace the same way
         # document.cookie parses whitespace.
-        # ("  =  b  ;  ;  =  ;   c  =  ;  ", {"": "b", "c": ""}),
+        ("  =  b  ;  ;  =  ;   c  =  ;  ", {"": "b", "c": ""}),
```

Comment:
Testing [this](https://github.com/encode/starlette/blob/master/starlette/requests.py#L55).
I don't know why this line was commented out before. But to meet the condition that the key and value are both empty, this parameter is the case we need.

Reply:
This may have been on purpose? When was introduced?

Reply:
No idea. The test and `cookie_parser` were introduced by this [PR](https://github.com/encode/starlette/pull/900).
It looks like these tests were inspired by tornado test suite ( [relevant commit](https://github.com/encode/starlette/pull/900/commits/b548645a33569af52613e6ec3fdaf4ff61f553b6) ).
And I didn't see any discussion related to this specific parameter. It was just commented out at the beginning.

Reply:
Just did more search in tornado's code and found [this](https://github.com/tornadoweb/tornado/blob/e761622abff3328c00065b803f4e807d5c72b1aa/tornado/test/httputil_test.py#L561). They did have tested the case.